spec/models/gadget_spec.rb

```diff
@@ -8,12 +8,25 @@
   it { is_expected.to be_versioned }
 
   describe "updates", versioning: true do
-    it "generates a version for updates to `name` attribute" do
+    it "generates a version for updates" do
       expect { gadget.update_attribute(:name, "Hammer") }.to(change { gadget.versions.size }.by(1))
     end
 
-    it "ignores for updates to `brand` attribute" do
-      expect { gadget.update_attribute(:brand, "Stanley") }.not_to(change { gadget.versions.size })
+    context "ignored via symbol" do
+      it "doesn't generate a version" do
+        expect { gadget.update_attribute(:brand, "Picard") }.not_to(change { gadget.versions.size })
+      end
+    end
+
+    context "ignored via Hash" do
+      it "generates a version when the ignored attribute isn't true" do
+        expect { gadget.update_attribute(:color, "Blue") }.to(change { gadget.versions.size }.by(1))
+        expect(gadget.versions.last.changeset.keys).to eq %w[color updated_at]
+      end
+
+      it "doesn't generate a version when the ignored attribute is true" do
```

Comment:
This test fails.  This should succeed right?

Reply:
Hi @jaredbeck, so when using `ignored` with a Hash, it still creates a version for the timestamp.

Reply:
> This test fails. This should succeed right?

Yeah, I think so. Nice test. 👍